backend/app/notify/telegram_test.go

```diff
@@ -166,6 +166,26 @@ func TestTelegram_Send(t *testing.T) {
 		`\"\u003ci\u003esome parent text with a \u003ca href=\"http://example.org\"\u003elink\u003c/a\u003e and special text:\u0026amp; \u0026lt; \u0026gt; \u0026amp;\u003c/i\u003e\"\n\n`+
 		`↦  \u003ca href=\"http://example.org/\"\u003e[test title]\u003c/a\u003e","parse_mode":"HTML"}`,
 		string(res))
+
+	// special case for text with h1-h3 header https://github.com/umputun/remark42/issues/1202
+	c.Text = "<h1>Hello</h1>"
+	res, err = buildMessage(Request{Comment: c, parent: cp})
```

Comment:
I propose to omit parent comment, so the text would be relevant only to what we are testing, and also, you can combine both cases into a single test.